controller/trackerquery_test.go

```diff
@@ -106,11 +100,28 @@ func TestUpdateTrackerQuery(t *testing.T) {
 		t.Errorf("Id should be %s, but is %s", tqresult.ID, tqr.ID)
 	}
 
+	spaceType := "spaces"
```

Comment:
shouldn't you make a constant out of this variable ? I see it is also used in the `getCreateTrackerQueryPayload` function below.